app/src/main/java/org/breezyweather/domain/weather/index/PollutantIndex.kt

```diff
@@ -25,11 +25,12 @@ import kotlin.math.roundToInt
 enum class PollutantIndex(
     val id: String,
     val thresholds: List<Int>,
+    val molecularMass: Double?,
 ) {
-    O3("o3", listOf(0, 50, 100, 160, 240, 480)), // Plume 2023
-    NO2("no2", listOf(0, 10, 25, 200, 400, 1000)), // Plume 2023
-    PM10("pm10", listOf(0, 15, 45, 80, 160, 400)), // Plume 2023
-    PM25("pm25", listOf(0, 5, 15, 30, 60, 150)), // Plume 2023
+    O3("o3", listOf(0, 50, 100, 160, 240, 480), 48.0), // Plume 2023
+    NO2("no2", listOf(0, 10, 25, 200, 400, 1000), 46.0055), // Plume 2023
+    PM10("pm10", listOf(0, 15, 45, 80, 160, 400), null), // Plume 2023
+    PM25("pm25", listOf(0, 5, 15, 30, 60, 150), null), // Plume 2023
```

Comment:
You can include the molecular mass of these two, in case it is used by another source, or as an unit in the future ;)

Reply:
Unfortunately that's not possible. 😅

The molecular mass is (roughly) the sum of the atomic weights of all the atoms making up a molecule. For example, O₃ has a molecular mass of 48 because an oxygen atom has an atomic weight of 16. So three of them add up to 48.

There is not a single molecule that makes up PM2.5 or PM10. They are "collective terms" of different kinds of particles below 2.5µm and 10µm in diameter respectively. Hence there can be no molecular masses assigned to these two pollutant categories.

Reply:
Ah yes, you anticipated the case and added `if (pollutant.molecularMass == null) return null` in the function, alright, thanks! ;)